WalletWasabi.Backend/Global.cs

```diff
@@ -242,6 +243,11 @@ protected virtual void Dispose(bool disposing)
 		{
 			if (disposing)
 			{
+				if (WabiSabiCoordinator is { } wabiSabiCoordinator)
```

Comment:
Nit:

We can easily make `WabiSabiCoordinator` not nullable (not sure why it is in the first place), then we don't need this check.

Reply:
To make `WabiSabiCoordinator` not nullable requires a refactoring that would make this PR much larger and hard to review. 

> not sure why it is in the first place

Because it was the easiest thing to do. Designing correctly is much harder.